presets/onelogin/events/version.yaml

```diff
@@ -0,0 +1,4 @@
+versions:
+  - versionNumber: 1
+    changes: Initial commit of the OneLogin preset (silver only)
+    commit: 9bc30afe24ad3337d6adf498a77bd7f77fffbacc
```

Comment:
TODO: will need to follow this with another PR to update the commit ID once we have one